DataGenerator/src/server.js

```diff
@@ -43,13 +43,13 @@ const SERVER = NET.createServer((socket) => {
 
   // Pack and send a buffer at half second intervals
   interval = setInterval(() => {
-    buffOffset = 4; // Offset when adding each value to buf1
+    buffOffset = 5; // Offset when adding each value to buf1
 
     // const time = DateTime.now().minus(30_000); // thirty seconds in the past
     const time = DateTime.now();
     // console.log(time);
     //write start symbol
-    buf1.write("<bl>")
+    buf1.write("<bsr>")
```

Comment:
Consider adding constants for the header/footer strings and getting their length instead of using magic numbers